change/@internal-react-components-cbda6e4f-8622-4f48-bcc9-42d5fe413676.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "patch",
+  "comment": {
```

Comment:
I think you missed adding a comment here?

Reply:
done